python/pyarrow/tests/test_parquet.py

```diff
@@ -641,6 +641,28 @@ def make_sample_file(table_or_df):
     return pq.ParquetFile(buf)
 
 
+def test_byte_stream_split():
+    # This is only a smoke test.
```

Comment:
Right. Is it easily doable to check column encodings of the generated file?

Reply:
After reading through the code, it is possible to check what information was propagated to the ColumnWriter but not what encoder object was created. That happens in the callstack
TypedColumnWriterImpl
    MakeEncoder <------ this is where the final decision happens based on dictionary enablement, byte_stream_split_enablement, type.

I am not able to find that one out without modifying the API and maybe it's not something which is necessary to check? I suppose it's up to the implementation to decide what it will use in the end as long as the parquet file is valid and results are ok.

Reply:
So, TLDR: not easily doable according to me.

Reply:
Fair enough.